src/universal/routes.jsx

```diff
@@ -22,11 +22,17 @@ import React from 'react';
 import ModuleRoute from 'holocron-module-route';
 import { Route } from '@americanexpress/one-app-router';
 import { applicationError, clearError } from '@americanexpress/one-app-ducks';
+import { getModule } from 'holocron';
+import hasChildRoutes from './utils/hasChildRoutes';
 
 const createRoutes = (store) => {
   const rootModuleName = store.getState().getIn(['config', 'rootModuleName']);
+  const RootModule = getModule(rootModuleName);
+  const rootHasChildRoutes = hasChildRoutes(RootModule);
+
   return [
-    <ModuleRoute moduleName={rootModuleName} store={store} />,
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <ModuleRoute moduleName={rootModuleName} store={store} {...(!rootHasChildRoutes && { path: '/' })} />,
```

Comment:
Not sure if this is better?
`<ModuleRoute moduleName={rootModuleName} store={store} path={rootHasChildRoutes ? undefined : '/'} />`

Reply:
the latter is more clear I feel